[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 8] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 8] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 8 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 8] Sesch channel 8 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 8] EOF received (8)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 8] EOF sent (8)
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64665'), (b'X-Server-Time', b'1693274810'), (b'Date', b'Tue, 29 Aug 2023 02:06:50 GMT'), (b'Content-Length', b'1392')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/staking/v1beta1/region "HTTP/1.1 200 OK"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 9] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 9] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 9 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 9] Sesch channel 9 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 9] EOF received (9)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 9] EOF sent (9)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 10] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 10] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 10 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 10] Sesch channel 10 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 10] EOF received (10)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 10] EOF sent (10)
[35mDEBUG   [0m httpcore.connection:_trace.py:45 close.started
[35mDEBUG   [0m httpcore.connection:_trace.py:45 close.complete
[35mDEBUG   [0m httpcore.connection:_trace.py:45 connect_tcp.started host='192.168.0.207' port=1317 local_address=None timeout=10 socket_options=None
[35mDEBUG   [0m httpcore.connection:_trace.py:45 connect_tcp.complete return_value=<httpcore._backends.sync.SyncStream object at 0x7f3155cfcca0>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64667'), (b'X-Server-Time', b'1693274821'), (b'Date', b'Tue, 29 Aug 2023 02:07:01 GMT'), (b'Transfer-Encoding', b'chunked')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/D3A2E700308D8ABF8CD00F7CD92C97A3BEAFFD603C6949169D132E7A6960B8F0 "HTTP/1.1 200 OK"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 11] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 11] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 11 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 11] Sesch channel 11 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 11] EOF received (11)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 11] EOF sent (11)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 12] Max packet in: 32768 bytes
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 12] Max packet out: 32768 bytes
[35mDEBUG   [0m paramiko.transport:transport.py:1893 Secsh channel 12 opened.
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 12] Sesch channel 12 request ok
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 12] EOF received (12)
[35mDEBUG   [0m paramiko.transport:channel.py:1210 [chan 12] EOF sent (12)
[35mDEBUG   [0m httpcore.connection:_trace.py:45 close.started
[35mDEBUG   [0m httpcore.connection:_trace.py:45 close.complete
[35mDEBUG   [0m httpcore.connection:_trace.py:45 connect_tcp.started host='192.168.0.207' port=1317 local_address=None timeout=10 socket_options=None
[35mDEBUG   [0m httpcore.connection:_trace.py:45 connect_tcp.complete return_value=<httpcore._backends.sync.SyncStream object at 0x7f31581dd7e0>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64669'), (b'X-Server-Time', b'1693274831'), (b'Date', b'Tue, 29 Aug 2023 02:07:11 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64670'), (b'X-Server-Time', b'1693274834'), (b'Date', b'Tue, 29 Aug 2023 02:07:14 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64670'), (b'X-Server-Time', b'1693274837'), (b'Date', b'Tue, 29 Aug 2023 02:07:17 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64671'), (b'X-Server-Time', b'1693274840'), (b'Date', b'Tue, 29 Aug 2023 02:07:20 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64671'), (b'X-Server-Time', b'1693274843'), (b'Date', b'Tue, 29 Aug 2023 02:07:23 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64672'), (b'X-Server-Time', b'1693274846'), (b'Date', b'Tue, 29 Aug 2023 02:07:26 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64673'), (b'X-Server-Time', b'1693274849'), (b'Date', b'Tue, 29 Aug 2023 02:07:29 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64673'), (b'X-Server-Time', b'1693274852'), (b'Date', b'Tue, 29 Aug 2023 02:07:32 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64673'), (b'X-Server-Time', b'1693274855'), (b'Date', b'Tue, 29 Aug 2023 02:07:35 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_headers.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 send_request_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_headers.complete return_value=(b'HTTP/1.1', 404, b'Not Found', [(b'Content-Type', b'application/json'), (b'Grpc-Metadata-Content-Type', b'application/grpc+cosmos-sdk-grpc-codec'), (b'Grpc-Metadata-X-Cosmos-Block-Height', b'64673'), (b'X-Server-Time', b'1693274858'), (b'Date', b'Tue, 29 Aug 2023 02:07:38 GMT'), (b'Content-Length', b'114')])
[32mINFO    [0m httpx:_client.py:1013 HTTP Request: GET http://192.168.0.207:1317/cosmos/tx/v1beta1/txs/5D5DA86830EDF894D169A583388FFD40A6A7C81B6A508292396303AE0A9848AC "HTTP/1.1 404 Not Found"
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.started request=<Request [b'GET']>
[35mDEBUG   [0m httpcore.http11:_trace.py:45 receive_response_body.complete
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.started
[35mDEBUG   [0m httpcore.http11:_trace.py:45 response_closed.complete